### ISSUE TITLE
Why use K.conv2d ?

### ISSUE THREAD

**Issue by mpariente** [keunwoochoi/kapre#29]
I wonder why do you use keras.conv2d in `Spectrogram` knowing that the STFT computation is based on a 1d convolution. 
If you would use keras.conv1d, I think you would be able to remove one axis from the filters and the inputs and thus gain in computation efficiency. 
Do you have any argument why you use 2D convolutions over 1D convolutions? Is it about the shape ?

**Comment by mpariente** [comment]
Ok, I just saw that in tensorflow `tf.nn.conv1d` reshapes its inputs and calls `tf.nn.conv2d` so, indeed, it's probably better to use `K.conv2d`, at least with tensorflow backend.

**Comment by keunwoochoi** [comment]
Exactly!
